src/fontra/views/fontoverview/fontoverview.js

```diff
@@ -340,6 +346,12 @@ export class FontOverviewController extends ViewController {
     const glyphItemList = this.glyphOrganizer.filterGlyphs(combinedGlyphItemList);
     const glyphSections = this.glyphOrganizer.groupGlyphs(glyphItemList);
     this.glyphCellView.setGlyphSections(glyphSections);
+
+    // Show placeholder if no glyphs are found
+    const fontOverviewNoGlyphsContainer = document.querySelector(
+      "#font-overview-no-glyphs"
+    );
+    fontOverviewNoGlyphsContainer.classList.toggle("hidden", glyphSections.length);
```

Comment:
Am I missing something? I axpected this to be `!glyphSections.length`

Reply:
By default is is hidden, so if there is a glyphs, it will remove it. Ah, maybe here is the misunderstanding. Maybe better show it by default and only hide it if there are glyphs. Let me change it.

Reply:
The argument to `.toggle()` is "on/off", so `.toggle("xxx", true)` will _add_ the class.

Reply:
If `glyphSections.length` equal 0 -> `false`
and if `glyphSections.length` greater 0 -> `true`
Then add `hidden` if `true` which is `glyphSections.length` greater 0. 
So, it's correct. Hide it if there are glyphs and don't hide it if there are no glyphs.
Anyway, I tested it and it works like expected.

Reply:
Proof:
https://github.com/user-attachments/assets/872bf567-7f03-4c18-ae3c-0aa2e722d20b

Reply:
You're absolutely right. It's late here... Let's make the condition explicit by adding `> 0`.

Reply:
OK. Done.